### DISCUSSION TITLE
I wanted to start a new web dev project, need your feedback .

### DISCUSSION THREAD

**Discussion by Namanmittal0007** [EddieHubCommunity/support#250] [Ideas]
I am starting a new Path Finding Visualizer and want to showcase this project on my resume.
I know the basis of HTML, CSS wanted to know what all I'll be needed to learn beforehand to start with the project.
Would love to collaborate and learn with you all in this project.
Project idea:
https://www.youtube.com/watch?v=n4t_-NjY_Sg&t=469s ( 02:54 - 08:14)
I know the algorithms used behind the project but new to Web development.

**Comment by AllanRegush** [comment]
I would go in any of these directions.
1. make a grid of divs then style as squares
2. use html canvas https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API

**Reply by NirmalSilwal** [reply]
@Namanmittal0007  I am also thinking to make similar visualization project next month. Currently I am learning [Data structures](https://github.com/NirmalSilwal/Data-Structure-and-Algorithm-Java-interview-kit).
If you are doing in months, we can collaborate. I also know basic HTML, CSS stuffs.

Even we can share ideas if our time frame don't match. What do you think?

**Reply by Namanmittal0007** [reply]
@NirmalSilwal All-time ready for collabs :)

**Reply by NirmalSilwal** [reply]
thanks @Namanmittal0007
